contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -146,7 +148,7 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
   // s_coordinatorLinkPremiumPercentage is the premium ratio in percentage for link payment. For example, a
   // value of 0 indicates no premium. A value of 15 indicates a 15 percent premium.
   // Wrapper has no premium. This premium is for VRFCoordinator.
-  uint8 private s_coordinatorLinkPremiumPercentage;
+  uint8 public s_coordinatorLinkPremiumPercentage;
```

Comment:
Why is this suddenly changed to `public`

Reply:
Ah, probably by mistake... Fixed it now, good catch!